server_config.coffee

```diff
@@ -85,6 +85,8 @@ config.salt = process.env.COCO_SALT or 'pepper'
 config.cookie_secret = process.env.COCO_COOKIE_SECRET or 'chips ahoy'
 
 config.isProduction = config.mongo.host isnt 'localhost'
+config.mainHostname = process.env.COCO_MAIN_HOSTNAME or 'localhost'
+config.unsafeContentHostname = process.env.COCO_UNSAFE_CONTENT_HOSTNAME or 'localhost'
```

Comment:
Not sure what the most meaningful defaults are here. In some places, "host" includes port number, so these don't work if you fake `isProduction` but are still on port 3000